client/src/pages/Doctors.tsx

```diff
@@ -16,11 +16,17 @@ const Doctors = () => {
   }, []);
 
   return (
-    <Stack spacing={3} direction="column">
+    <Box
+      display="flex"
+      justifyContent="center"
+      flexDirection="column"
+      alignItems="center"
+      sx={{ width: { xs: '100%', sm: '50%' }, margin: ' 40px auto' }}
+    >
       {doctors.map((doctor: IUser) => (
-        <Doctor key={doctor.id} user={doctor} />
+        <Doctor isEmergency={false} key={doctor.id} user={doctor} />
```

Comment:
You had to do this because you didnt seperate your components properly
Read: make your components precise and dont do much things:
https://beta.reactjs.org/learn/keeping-components-pure